packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -1528,8 +1528,38 @@ def _is_timeout_expired(self) -> bool:
             0, self._timeout
         )
 
+    def _get_reset_params(self, default: bool) -> Optional[List[Tuple[str, str]]]:
+        """Get the parameters for a hard reset request to Tendermint."""
+        if default:
+            return None
+
+        last_round_transition_timestamp = (
+            self.context.state.round_sequence.last_round_transition_timestamp
+        )
+        genesis_time = last_round_transition_timestamp.astimezone(pytz.UTC).strftime(
+            GENESIS_TIME_FMT
+        )
+        # Initial height needs to account for the asynchrony among agents.
+        # For that reason, we are using an offset in the initial block's height.
+        # The bigger the observation interval, the larger the lag among the agents might be.
+        # Also, if the observation interval is too tiny, we do not want the offset to be too small.
+        # Therefore, we choose between a minimum value and the interval multiplied by a constant (< 1 suggested).
+        initial_height = str(
+            self.context.state.round_sequence.last_round_transition_tm_height
+            + max(
+                MIN_HEIGHT_OFFSET,
+                math.ceil(self.params.observation_interval * HEIGHT_OFFSET_MULTIPLIER),
+            )
+        )
```

Comment:
there is an issue to address this: https://github.com/valory-xyz/open-autonomy/issues/879